pkg/client/unversioned/record/event_test.go

```diff
@@ -286,24 +289,24 @@ func TestEventf(t *testing.T) {
 				if !item.expectUpdate {
 					t.Errorf("Expected event create(), got event update()")
 				}
-				called <- struct{}{}
+				wg.Done()
 				return returnEvent, nil
 			},
 		}
 		eventBroadcaster := NewBroadcaster()
 		sinkWatcher := eventBroadcaster.StartRecordingToSink(&testEvents)
 		logWatcher1 := eventBroadcaster.StartLogging(t.Logf) // Prove that it is useful
+		wg.Add(1)
```

Comment:
The idea is to keep each `Add(1)` close to what will later trigger the `Done()` call. If you do a single `Add(n)` at the top, it might later be harder to spot what blocks of code amount for how many of the `Done()` calls.

This of course doesn't apply to loops.